FFXIVClientStructs/FFXIV/Client/Game/BGMSystem.cs

```diff
@@ -15,10 +15,10 @@ public unsafe partial struct BGMSystem {
     [FieldOffset(0xDC)] public bool PlayBattleBGM; // is set in update loop, based on combat
 
     [FieldOffset(0xE0)] private float UnkBattleBGMDuration0;
-    [FieldOffset(0xe4)] private float UnkBattleBGMDuration1;
+    [FieldOffset(0xE4)] private float UnkBattleBGMDuration1;
 
-    [FieldOffset(0xF8)] public bool LastingBGM;
-    [FieldOffset(0xF9)] public bool ContinueBGMUntilWarp;
+    [FieldOffset(0xF8)] public bool EnableLastingBGM;
+    [FieldOffset(0xF9)] public bool EnableContinueBGMUntilWarp;
```

Comment:
Rename the function and not the fields. This is as people will be confused why fields are named `Enable`

Reply:
Denied.